gsy_myco_sdk/matching_algorithms/preferred_partners_algorithm.py

```diff
@@ -124,6 +70,8 @@ def _can_order_be_matched(
 
         bid_required_energy, bid_required_clearing_rate = (
             cls._get_required_energy_and_rate_from_order(bid, bid_requirement))
+        bid_requirement = deepcopy(bid_requirement)
+        bid_requirement.pop("original_price", None)
```

Comment:
These two lines will be deleted once we deprecate the `original_price` of the requirements